### ISSUE TITLE
Fullscreen goes to windowed mode after alt-tabbing out and back

### ISSUE THREAD

**Issue by spunky44** [MonoGame/MonoGame#5105]
In XNA, you could alt-tab out of a full screen game, and when returning, it would still be full screen.
In MonoGame, when you open the game again, it goes to windowed mode.

It would be nice to have this fixed for Windows at least.

**Comment by ShawnCowles** [comment]
I managed to get a decent solution working through setting `Graphics.HardwareModeSwitch= false;` before starting full screen mode.

I'm not sure if you're still looking for an answer, but maybe it will help the next one who finds this bug.

**Comment by Jjagg** [comment]
Fixed for DX in #5585
